### ISSUE TITLE
R package

### ISSUE THREAD

**Issue by VLucet** [WildCoLab/Introduction-to-Camera-Trap-Data-Management-and-Analysis-in-R#15]
Hi! Really enjoying the content of the course. I was wondering if there had ever been any talk of turning the content of this course into a R package for camera trap data exploration, and whether you'd welcome this kind of idea. I can develop my thinking behind such a project if that is welcome. In any case, the course is phenomenal and extremely valuable as it is.

**Comment by ChrisBeirne** [comment]
Hi Val, this is a great idea! Perhaps we should have a call to think about this - might be up @mabecker89 's street too. @wildcole is on sabbatical  but it sounds like something he would be interested in!

**Comment by VLucet** [comment]
Cool! There is some nice synergy to create with @mabecker89's wildrtrax, building up to a camera-trap-verse (CTverse?) set up for standardizing CT analysis in R. Happy to join a call of course.

**Comment by mabecker89** [comment]
Hey guys, yes, for sure I'd definitely be interested. Chris and I chatted about something like this a few years ago and I think it's still a great idea. Ideally wildRtrax would just import a package like that (with maybe some convenience functions to help things play nice with WildTrax-formatted data?). Happy to jump on a call anytime!

**Comment by wildcole** [comment]
Yes I agree it is a good idea and something we should support / work on
